### ISSUE TITLE
sum over properties

### ISSUE THREAD

**Issue by agoscinski** [metatensor/metatensor#376]
Is there a way to do sum over properties? Would be handy for aggregation in kernels, not high priority.

**Comment by Luthaf** [comment]
I don't see what you mean by "aggregation in kernels"? Computing the kernels from two feature vectors? Or something different?

**Comment by agoscinski** [comment]
I mean the sum/mean over central atoms with aggregation. In a full kernel there are centers in the property axis which one want to aggregate. For the sparse kernel this is not really important, because one only has to do it over the one axis. But there are other use cases where this might be useful.

**Comment by Luthaf** [comment]
Right, for going from a per-atom full kernel to a per-structure full kernel. That sounds good!
